pyaedt/application/Design.py

```diff
@@ -3916,3 +3916,32 @@ def set_temporary_directory(self, temp_dir_path):
         """
         self.odesktop.SetTempDirectory(temp_dir_path)
         return True
+
+    @pyaedt_function_handler()
+    def design_settings(self):
+        """Get design settings for the current AEDT app.
+
+        Returns
+        -------
+        dict
+            Dictionary of valid design settings.
+
+        References
+        ----------
+
+        >>> oDesign.GetChildObject("Design Settings")
+        """
+        try:
+            design_settings = self._odesign.GetChildObject("Design Settings")
+        except Exception:
+            raise Exception("Could not use GetChildObject('Design Settings')")
```

Comment:
@nnetake return False if the exception is raised. 
Try to cover this line as well with the code and fix the returns in the docstring so if it fails return False.

Reply:
Done. thanks!